solr/solrj/src/java/org/apache/solr/client/solrj/request/DocumentAnalysisRequest.java

```diff
@@ -85,7 +86,7 @@ protected DocumentAnalysisResponse createResponse(SolrClient client) {
   }
 
   @Override
-  public ModifiableSolrParams getParams() {
+  public SolrParams getParams() {
```

Comment:
With new clarity from the Javadocs, we should never declare the return type as ModifiableSolrParams unless we have a consistent instance to return.